src/server/routes/v2/github.ts

```diff
@@ -0,0 +1,62 @@
+import { FastifyInstance } from "fastify";
+import { ManifestModel, addOrUpdatePackage } from "../../../database";
+import { validateApiToken } from "../../helpers";
+import ghService from "../../ghService/index";
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  // *----------------- import package update --------------------
+  fastify.get("/import", { onRequest: validateApiToken }, async () => {
+    const yamls = await ghService.initialPackageImport();
+    console.log(`yamls length - ${yamls.length}`);
+
+    const batchSize = yamls.length % 2 === 0 ? 150 : 155;
+    let batchIndex = 0;
+
+    while (batchIndex < yamls.length) {
+      const batch = yamls.slice(batchIndex, batchIndex + batchSize);
+
+      for (let i = 0; i < batch.length; i += 1) {
+        const pkg = (batch[i] as unknown) as ManifestModel;
+        console.log(`${pkg.Id} / ${pkg.Version} - ${batchIndex}`);
+
+        // eslint-disable-next-line no-await-in-loop
+        await addOrUpdatePackage(pkg);
```

Comment:
currently this will do the same thing as a single for loop going through all the packages 1 by 1 cos its awaiting for each one, what you probably want is Promise.all on the whole batch

same thing with the route below